kivymd/vendor/circularTimePicker/__init__.py

```diff
@@ -491,8 +489,7 @@ def number_at_pos(self, x, y):
 
 
 class CircularMinutePicker(CircularNumberPicker):
-    """:class:`CircularNumberPicker` implementation for minutes.
-    """
+    """:class:`CircularNumberPicker` implementation for minutes."""
 
     def __init__(self, **kw):
         super().__init__(**kw)
```

Comment:
Updated by pre-commit of github actions. 8994a39cd2b1050aa6b8342b533d148cbe2e065a